src/components/Movements/Movements.columns.js

```diff
@@ -37,13 +42,16 @@ const getColumns = ({
       )
     },
     copyText: rowIndex => filteredData[rowIndex].id,
-  },
+  }] : []),
```

Comment:
isn't this check supposed to return an object?

Reply:
nope, it's intended to return an empty array here